src/coreclr/vm/binder.h

```diff
@@ -65,7 +65,6 @@ enum BinderClassID
     CLASS__SINGLE   = CLASS__ELEMENT_TYPE_R4,
     CLASS__DOUBLE   = CLASS__ELEMENT_TYPE_R8,
     CLASS__STRING   = CLASS__ELEMENT_TYPE_STRING,
-    CLASS__TYPED_REFERENCE = CLASS__ELEMENT_TYPE_TYPEDBYREF,
```

Comment:
I think this should stay. Instead, the line in corelib.h should be `// DEFINE_CLASS(TYPED_REFERENCE...`, similar to https://github.com/dotnet/runtime/blob/acbb049afbf6d22a851459f58c84e4a115b7f542/src/coreclr/vm/corelib.h#L796